.github/workflows/e2e-matrix.yml

```diff
@@ -108,6 +131,10 @@ jobs:
 
     - name: Run tests
       run: |
+        # This is a hack to make sure binary in /bin from the base image are not picked by default
+        # On ubuntu-arm, there is a /bin/go that shadows the one setup-go installs
+        export PATH="$(echo "$PATH" | sed -e 's/^\/bin://'):/bin"
+        export KO_DEFAULTPLATFORMS=linux/$(go env GOARCH)
```

Comment:
This is required to enforce `ko` builds the "host" arch (aka arm64 on arm64 node, ...)